src/main/org/pantsbuild/jarjar/util/JarProcessor.java

```diff
@@ -23,12 +23,13 @@
     /**
      * Process the entry (p.ex. rename the file)
      * <p>
-     * Returns <code>true</code> if the processor has has changed the entry. In this case, the entry can be removed
-     * from the jar file in a future time. Return <code>false</code> for the entries which do not have been changed and
-     * there fore are not to be deleted
+     * Returns <code>true</code> if the processor "successfully" processed the entry. In practice,
+     * "true" is used to indicate that the entry should be kept, and false is used to indicate the
+     * entry should be thrown away, and doesn't indicate anything in particular about what this
+     * JarProcessor actually did to the entry (if anything).
      *
-     * @param struct
```

Comment:
Nope, but there sure are some weird hardly-ever used keywords like `goto` and `strictfp` http://docs.oracle.com/javase/tutorial/java/nutsandbolts/_keywords.html